types.proto

```diff
@@ -16,15 +16,15 @@ message ModuleInput {
 	//  - only Ethereum network (EL and CL nodes) will be launched. Nothing else (no transaction spammer)
 	//  - params for the CL nodes will be ignored (e.g. CL node image, CL node extra params)
 	// This is a hack - it's not very elegant - but this is a commonly-requested feature
-	// The longterm solution is making the module trivial to decompose so we don't need flags like this; we're working
+	// The longterm solution is making the package trivial to decompose so we don't need flags like this; we're working
```

Comment:
should we rename the message name also `ModuleInput` ? or is this the expect name in APIC?

Reply:
Its set to that in APIC, Guillaume will be removing proto completely.

Reply:
ah! ok, thanks